<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calibration</class>
 <widget class="QDialog" name="Calibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compass Calibration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Magnetic Flux Density Y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_x_cur">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_y_cur">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Magnetic Flux Density Z:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_z_cur">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Magnetic Flux Density X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Current</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Step 1:&lt;/span&gt; Remove Old Calibration&lt;br/&gt;&lt;br/&gt;To recalibrate the Bricklet you first have to remove the old calibration.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="QPushButton" name="button_calibration_remove">
       <property name="text">
        <string>Remove Old Calibration</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="4">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Step 2:&lt;/span&gt; Calibration&lt;br/&gt;&lt;br/&gt;Move Compass Bricklet into every possible position, until the min/max values don't change anymore. Make sure that there are no magnetic sources in the vicinity of the Bricklet during the calibration.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="4">
      <widget class="QPushButton" name="button_calibration_save">
       <property name="text">
        <string>Save Calibration</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_x_min">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_y_min">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_z_min">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_x_max">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_y_max">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_z_max">
       <property name="text">
        <string>TBD</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="4">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="4">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="4">
      <widget class="QPushButton" name="button_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
